docs/backstop.md

```diff
@@ -0,0 +1,65 @@
+# BackstopJS Getting Started Guide
+
+## Overview
+
+BackstopJS is currently optional in this project and is used for visual regression testing to ensure consistency in the UI. To get started with BackstopJS, follow the steps below. The official guide can be found at the [BackstopJS GitHub Repository](https://github.com/garris/BackstopJS).
+
+## Goal
+
+Eventually BackstopJS will be implemented in the pipeline process to help catch errors before they make their way to higher environments.
+
+## Prerequisites
+
+Before running BackstopJS, ensure the following:
+
+- Both the API and Admin projects must be running.
+- In the .env file of the API project, make sure the following environment variables are uncommented and set to matching values:
+  - `NOTIFY_E2E_TEST_EMAIL`
+  - `NOTIFY_E2E_TEST_PASSWORD`
```

Comment:
These need to be set for _both_ the API and the Admin!  Otherwise the tests won't work.

In other words, the Admin site needs to be running in E2E test mode, which is enabled by having both of these values set (and matching what's been defined for the API).

Reply:
```suggestion
- In the .env file of both the API and Admin projects, make sure the following environment variables are uncommented and set to matching values:
  - `NOTIFY_E2E_TEST_EMAIL`
  - `NOTIFY_E2E_TEST_PASSWORD`
```